### DISCUSSION TITLE
Making games without sfml

### DISCUSSION THREAD

**Discussion by 3573577** [community/community#116772] [Programming Help] [answered]
### Body

How to make game board made up of "#"(20×17) and how to move player denoted by "%" left right up down using user input without sfml 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by 3573577** [comment, marked as answer]
Please give answer

**Comment by abphir** [comment]
This approach can be used to create simple [maze games, checkers, or other grid-based games](https://worldapkbox.com) that emphasize logical thinking and problem-solving.
Creating a game board in Python similar to the WorldBox God Simulator **worldapkbox.com** 
**Setup the Game Board:**
Create a 20x17 grid of "#".
Place the player "%" at (0, 0).
Display the Game Board:
Function to print the board.
**Move the Player:**
User inputs: 'w' (up), 's' (down), 'a' (left), 'd' (right).
Update position within bounds.
**Example Code:**
```python
def create_board(rows, cols):
    return [["#" for _ in range(cols)] for _ in range(rows)]
def print_board(board, player_pos):
    board[player_pos[0]][player_pos[1]] = "%"
    for row in board:
        print("".join(row))
def move_player(player_pos, move, rows, cols):
    x, y = player_pos
    if move == 'w' and x > 0: x -= 1
    elif move == 's' and x < rows - 1: x += 1
    elif move == 'a' and y > 0: y -= 1
    elif move == 'd' and y < cols - 1: y += 1
    return x, y
rows, cols = 20, 17
board = create_board(rows, cols)
player_pos = (0, 0)
while True:
    print_board(board, player_pos)
    move = input("Enter move (w/a/s/d): ").strip().lower()
    if move in ['w', 'a', 's', 'd']:
        player_pos = move_player(player_pos, move, rows, cols)
```
This setup can create simple games like mazes and checkers.
